Documentation/Signoffs/detaildesgincamera.md

```diff
@@ -0,0 +1,137 @@
+# Detailed Design for Camera Subsystem within Primary Data Acquisition System
+
+## Big Picture
+
+The primary data acquisition system will utilize real-time camera sensors to collect images of sectors of parking spots in the lot behind Bell Hall, in which these images will then be transmitted via wireless communication to a server where the images will be processed by an artificial intelligence (AI) algorithm to determine the number of Red parking spots that are filled and unfilled by cars. This subsystem is a critical subsystem because these cameras will provide the primary data to the system that will determine the count of vehicles in a lot. The images from these cameras will be sent over to the server to be processed by an artificial intelligence (AI) algorithm to determine the number of available parking spots in the Bell Hall lot. This number will then be cross-checked with that derived from the secondary acquisition system to determine its accuracy.
+
+## Specifications
+
+1. Mounting
+    * Attachment location
+        * Light posts located around the parking lot (see Figure 2)
+        * Light posts are 25 feet tall
+    * Not allowed to drill into the posts
+        * Hose clamp for block (mounting surface) to top of light pole
+        * More refined mount will be designed by ME Team
+    * Not allowed to add additional infrastructure (i.e. posts)
+2. Monitoring
+    * Cameras
+        * All parking spots must be seen by cameras
+        * Able to recognize car from feed
+        * Have wireless communication to server
+3. Maintainability
+    * Should require minimum maintenance once installed
+4. Power
+    * All cameras will be solar powered to provide minimal maintenance
+5. Communication
+    * Wireless communication
+        * Bluetooth not an option due to range	
+        * WiFi
+            * Need consideration of following factors:
+                * Quality
+                * Range
+                * Downtimes
+
+## Analysis
+
+1. Mounting
+    * In order for the camera to operate correctly it will need to be mounted securely onto the light post. However, a constraint we received from facilities is that drilling holes in the pole is not allowed. Due to this we need a mounting method that does not require drilling and is adjustable in the event that we need to relocate the camera on the pole. For this reason we plan to use a hose clamp/vertical bracket combination similar to Dahua Technology pole mount bracket (MFR# PFA150). Currently we have not received pole diameter dimensions from facilities so the correct hose clamp size has not been determined. However, this is a minor issue that can quickly be resolved by either measuring the pole or receiving the information from facilities. Hose clamps have a wide size range that should satisfy our requirements.
```

Comment:
This is not analysis. This is continued constraints. Further, the mounting method must be represented in a buildable schematic. Probably a 3d modeling software.